spec/default_spec.rb

```diff
@@ -48,6 +49,7 @@
     it 'should config JAAS' do
       chef_run.converge(described_recipe)
       expect(chef_run).to create_template('/opt/confluent/confluent-2.0.1/jaas.conf')
+      # rubocop:disable Lint/AmbiguousBlockAssociation
```

Comment:
Did not realize thats what this was complaining about, fixed https://github.com/bbaugher/confluent/pull/24/commits/32cf7f244846c5c0eb98b20e7d2ca98dcfd7910e